### PR TITLE
bump gleam version to at least 1.1.0

### PR DESCRIPTION
**Pull request by TanklesXL**

### CODE REVIEW

**Comment by hayleigh-dot-dev** [comment]
Did you run `gleam format` on the entire codebase? I see there are some minor formatting changes, just wnated to make sure that was all of them.

**Comment by TanklesXL** [comment]
yep i ran it on the whole thing, if i hadnt the `format` job would have failed :D

**Comment by TanklesXL** [comment]
> Did you run `gleam format` on the entire codebase? I see there are some minor formatting changes, just wnated to make sure that was all of them.

yep i ran it on the whole thing, if i hadnt the `format` job would have failed :D

File: gleam.toml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 name = "code_review"
 version = "1.0.0"
 
+gleam = ">= 1.1.0 and < 2.0.0"
```
Comment (hayleigh-dot-dev) [inline gleam.toml:4]: What about the project requires 1.1.0?

Comment (lpil) [reply gleam.toml:4]: I'd probably no limit it to under 2

Comment (TanklesXL) [reply gleam.toml:4]: Hmmm technically there is nothing forcing this to only use gleam >= 1.1.0.  We can probably leave this one unchanged and I can definitely remove the less than 2.0 constraint

Comment (TanklesXL) [reply gleam.toml]: fixed :D thanks for the feedback folks

**Comment by bcpeinhardt** [comment]
lgtm

**Pull request merged**